presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/hdfs-site-overrides.xml

```diff
@@ -0,0 +1,6 @@
+<configuration>
+    <property>
+        <name>dfs.safemode.threshold.pct</name>
+        <value>0</value>
```

Comment:
Add a comemnt explaining what it does.

Also, why not add this in the hdp3.1-hive image?

Reply:
I was thinking about that but I guess it's much faster that way (the same question should apply to enabling acid using site-override)

Reply:
What is faster?

Note that changing hdp3.1-hive also improves startup time when this env is used by means of `confg-hdp3`.
And, there is no reason for singlenode-hdp3 to have different safe mode / startup time than singlenode with hdp3 config.

Reply:
Ok, I'll apply this to hdp3.1-hive image.

Reply:
https://github.com/prestosql/docker-images/pull/60

Reply:
Faster in introducing mitigation (adding to docker image requires PR + release + change in prestosql but I'm fine with that if you are).